### ISSUE TITLE
Movielens-25M dataset missing "ratings.csv" file

### ISSUE THREAD

**Issue by mengzaiqiao** [beta-team/beta-recsys#352]
### Describe the bug
 I tried to use the Movielens-25M dataset, but it doesn't work. It complains on missing "ratings.csv" file. The file is missing indeed and it's called ratings.dat in the archive. 

### To Reproduce
Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) here. We hope we can simply copy/paste/run it. It is also nice to share a hosted runnable script (e.g. Google Colab), especially for hardware-related problems.

### Describe your attempts
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this is not a duplicate issue

You should also provide code snippets you tried as a workaround, StackOverflow solution that you have walked through, or your best guess of the cause that you can't locate (e.g. cosmic radiation).

### Context
 - **OS** [e.g. Windows 10, macOS 10.14]: 
 - **Hardware** [e.g. CPU only, GTX 1080 Ti]: 
 - **Environment** [e.g. Pytorch 1.4, Tensorflow 2.0]:

In addition, figure out your used Beta-Recsys version. Then attach the corresponding label on the issue.

### Additional Information
Other things you want the developers to know.

**Comment by leungyukshing** [comment]
@mengzaiqiao I try to download MovieLens-25M both using the url in the code and from the website directly. These is NO `ratings.dat` files in the folder.
